settings.gradle

```
@@ -0,0 +1,44 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * The settings file is used to specify which projects to include in your build.
+ * For more detailed information on multi-project builds, please refer to https://docs.gradle.org/8.3/userguide/building_swift_projects.html in the Gradle documentation.
+ */
+
+pluginManagement {
+    plugins {
+        id "com.github.spotbugs-base" version "${spotbugsPluginVersion}"
+        id "com.github.johnrengelman.shadow" version "${shadowJarPluginVersion}"
+        id "de.undercouch.download" version "${downloadPluginVersion}"
+        id "net.researchgate.release" version "${releasePluginVersion}"
+        id "io.ballerina.plugin" version "${ballerinaGradlePluginVersion}"
```

IINM we only need the below 2 plugins as this is a pure ballerina project. I noticed this same issue in some already revamped connectors, which we should adress seperately.
```suggestion
        id "net.researchgate.release" version "${releasePluginVersion}"
        id "io.ballerina.plugin" version "${ballerinaGradlePluginVersion}"
```

---

Fixed with [57923b504763d296e6a2774f752731cbd69ea0a1](https://github.com/ballerina-platform/module-ballerinax-stripe/pull/1/commits/57923b504763d296e6a2774f752731cbd69ea0a1)